apps/desktop/electron-builder-snap.config.js

```
@@ -1,60 +1,20 @@
         'to': 'bin/bridge',
       },
     ],
-    'icon': 'build/static/images/icons/512x512.png',
+    'icon': 'app/build/static/images/icons/512x512.png',
     'artifactName': 'OneKey-Wallet-${version}-linux-${arch}.${ext}',
     'executableName': 'onekey-wallet',
     'category': 'Utility',
     'target': ['snap'],
-  },
-  'afterSign': 'scripts/notarize.js',
+  }
```

ESLint

[error] Insert `,`

(prettier/prettier)